lib/webinspector/index.js

```diff
@@ -29,6 +29,7 @@ const PARTIAL_MESSAGE_SUPPORT_DEPRECATION_VERSION = 11;
  * @property {boolean} verboseHexDump Turn on logging of _all_ communication as
  *                                    hex dump. Defaults to false
  * @property {*} socketClient The socket client where the communication will happen
+ * @param {number} maxFrameLength - The maxsize of frame size
```

Comment:
`@param {number} maxFrameLength [20 * 1024 * 1024] - The maximum size in bytes of a single data frame in the device communication protocol`

Reply:
same below